### ISSUE TITLE
Add lzo compression to hdf

### ISSUE THREAD

**Issue by blink1073** [metabolite-atlas/metatlas#2]
https://pytables.github.io/usersguide/optimization.html#searchoptim

**Comment by blink1073** [comment]
No compression:   Write:  1m37.428s   Read:  109ms   Size: 24Mb
ZLIB compression: Write: 7m1.434s     Read: 191ms     Size: 3.4Mb
LZO compression: Write: 3m0.839s      Read:  152ms   Size: 4.1Mb
LZO with indices:  Write:  3m5.066s     Read:   215ms    Size: 8.9Mb




Read test:

```python
fid = tables.open_file(out_file)
table = fid.root.spectra
%time ms_1 = [y['scan_time'] for y in table.where('(collision_energy < 1000)')]
```

Load code:

```python
FILTERS = tables.Filters(complib='lzo', complevel=1)
out_file = tables.open_file(out_file_name, "w", filters=FILTERS)

 table.cols.ms_level.create_index()
    table.cols.mz.create_index()
    table.cols.scan_time.create_index()
    table.cols.i.create_index()
    table.cols.polarity.create_index()
    table.cols.precursor_intensity.create_index()
    table.cols.collision_energy.create_index()
    table.cols.precursor_MZ.create_index()
```

**Comment by blink1073** [comment]
Based on these results, I'd say we should use LZO compression without creating indices for the columns.
Do you concur @benbowen?

**Comment by benbowen** [comment]
If not lzo then something that's already installed on Ubuntu and centos
Linux.

On Monday, March 23, 2015, Steven Silvester <notifications@github.com>
wrote:

> Based on these results, I'd say we should use LZO compression without
> creating indices for the columns.
> Do you concur @benbowen <https://github.com/benbowen>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metabolite-atlas/metatlas/issues/2#issuecomment-85314654>
> .
>


-- 
--------------
Email: BPBowen@lbl.gov
Google Voice: (510) 214-6683
Mobile: (480) 225-2341
MS 977-221, Lawrence Berkeley Lab
OpenMSI: http://openmsi.nersc.gov

**Comment by blink1073** [comment]
lzo was already installed on Ubuntu 14.04

**Comment by blink1073** [comment]
And I see it in the list of included packages of Centos 6.6.

**Comment by benbowen** [comment]
Perfect

On Monday, March 23, 2015, Steven Silvester <notifications@github.com>
wrote:

> And I see it in the list of included packages of Centos 6.6.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metabolite-atlas/metatlas/issues/2#issuecomment-85318216>
> .
>


-- 
--------------
Email: BPBowen@lbl.gov
Google Voice: (510) 214-6683
Mobile: (480) 225-2341
MS 977-221, Lawrence Berkeley Lab
OpenMSI: http://openmsi.nersc.gov

**Comment by blink1073** [comment]
Already implemented, closing.
